src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,8 @@
+import { useEffect, useState } from "react";
+import { useparams } from "react-router-dom";
+import { getArticleById } from "../utils/api";
+
+const SingleArticle = () => {
+    const [article, setArticle] = useState({});
+    const [isLoading, setIsLoading] = useState(true);
+}
```

Comment:
I won't review this - I assume you've gotten ahead into ticket 5 so I shouldn't be able see this as part of ticket 4